core/src/main/java/org/jboss/pnc/causeway/ctl/PncImportControllerImpl.java

```diff
@@ -210,6 +218,14 @@ private BuildImportResultRest importBuild(BuildRecordRest build, String username
             KojiImport kojiImport = translator.translate(nvr, build, artifacts, log, username);
             ImportFileGenerator importFiles = translator.getImportFiles(artifacts, log);
             buildResult = brewClient.importBuild(nvr, build.getId(), kojiImport, importFiles);
+
+            long artifactSize = artifacts.buildArtifacts.stream().map(pncArtifact -> pncArtifact.size).collect(Collectors.summingLong(i->i));
+            int artifactNumber = artifacts.buildArtifacts.size();
+
+            updateHistogram(metricsConfiguration, METRICS_ARTIFACTS_SIZE_KEY, artifactSize);
+            updateHistogram(metricsConfiguration, METRICS_ARTIFACTS_NUMBER_KEY, artifactNumber);
+            updateHistogram(metricsConfiguration, METRICS_LOGS_SIZE_KEY, log.length());
```

Comment:
This is not technically correct as some characters can have more than one byte in utf-8.

Reply:
@janinko  ok I calculated the lenght more precisely, falling back to normal String.lenght():
            int logLenght = log.length();
            try {
                logLenght= log.getBytes("UTF-8").length;
            } catch (UnsupportedEncodingException e) {
            }